app/components/ExternalIps.tsx

```diff
@@ -22,30 +22,25 @@ export function ExternalIps({ project, instance }: InstanceSelector) {
   if (isPending) return <SkeletonCell />
 
   const ips = data?.items
-    ? intersperse(
-        data.items.map((eip) => <IpLink ip={eip.ip} key={eip.ip} />),
-        <span className="text-quinary"> / </span>
-      )
-    : undefined
-
+  if (!ips || ips.length === 0) return <EmptyCell />
   return (
-    <div className="flex items-center gap-1 text-secondary">
-      {ips && ips.length > 0 ? ips : <EmptyCell />}
-      {/* If there's exactly one IP here, render a copy to clipboard button */}
-      {data?.items.length === 1 && <CopyToClipboard text={data.items[0].ip} />}
+    <div className="flex items-center gap-1">
+      {intersperse(
+        ips.map((eip) => (
+          <span className="flex items-center" key={eip.ip}>
+            <a
+              className="link-with-underline text-sans-semi-md"
+              href={`https://${eip.ip}`}
+              target="_blank"
+              rel="noreferrer"
+            >
+              {eip.ip}
+            </a>
+            <CopyToClipboard text={eip.ip} />
```

Comment:
I think this needs an `ml-1` to do what the `gap-1` was doing when this was at top level. I was afraid it would then look too close to the slash on the right, but it seems fine to me.

### before

<img width="194" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/117e3804-12d7-4504-9d3d-ea686651d6a3">


### after

<img width="198" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/06267818-caf1-4e1e-b25a-c25b7ef51c88">

Reply:
Yep +1